addon-test-support/@ember/test-helpers/setup-rendering-context.ts

```diff
@@ -13,14 +13,24 @@ import { hbs, TemplateFactory } from 'ember-cli-htmlbars';
 import getRootElement from './dom/get-root-element';
 import { Owner } from './build-owner';
 import getTestMetadata from './test-metadata';
-import { assert, deprecate } from '@ember/debug';
+import { assert } from '@ember/debug';
 import { runHooks } from './-internal/helper-hooks';
-import hasEmberVersion from './has-ember-version';
-import isComponent from './-internal/is-component';
-import { macroCondition, dependencySatisfies } from '@embroider/macros';
 import { ComponentRenderMap, SetUsage } from './setup-context';
 import { ensureSafeComponent } from '@embroider/util';
-import type { ComponentInstance } from '@glimmer/interfaces';
+import type {
+  ComponentInstance,
+  InternalComponentManager,
+} from '@glimmer/interfaces';
+import { importSync } from '@embroider/macros';
+
+let getInternalComponentManager: (
+  definition: object
+) => InternalComponentManager;
+
+// this would be much better as a regular `import` statement, but there really isn't any way to make typescript happy with thayar
+getInternalComponentManager =
+  //@ts-ignore
+  importSync('@glimmer/manager').getInternalComponentManager;
```

Comment:
(extremely non-blocking) We should see if we can write a type for `importSync` that makes this work. I'm not *sure* we can, but it's worth trying.

Reply:
Update, after poking a very little bit at it: I don’t think the language affordance is there. Alas.